src/config/section/compute.js

```diff
@@ -86,7 +86,7 @@ export default {
           dataView: true,
           groupAction: true,
           show: (record) => { return ['Stopped'].includes(record.state) },
-          args: ['podid', 'clusterid', 'hostid'],
+          args: ['podid', 'clusterid', 'hostid', 'bootintobios'],
```

Comment:
last args should be conditional on `hypervisor === vmware`. will be done after #231 has been merged with its [AutoGenView.vue changes](https://github.com/apache/cloudstack-primate/pull/231/files#diff-83f2f8976f46b2444d31fcd1895d8165R548)